src/Aquifer.Public.API/Endpoints/Resources/Search/GetResources/Endpoint.cs

```diff
@@ -86,19 +91,23 @@ private async Task<int> GetTotalResourceCountAsync(Request req, IQueryable<Resou
         return totalCount;
     }
 
-    private async Task<List<ResponseContent>> GetResourcesAsync(Request req,
+    private async Task<List<ResponseContent>> GetResourcesAsync(
+        Request req,
         IQueryable<ResourceContentVersionEntity> query,
         CancellationToken ct)
     {
-        var resources = await query.OrderBy(x => x.ResourceContent.Resource.EnglishLabel)
+        var languageCodeByIdMap = await _cachingLanguageService.GetLanguageCodeByIdMapAsync(ct);
+
+        var resources = await query
+            .OrderBy(x => x.ResourceContent.Resource.Id)
```

Comment:
This one line change is the biggest win in this PR.  However, it is a breaking change because the data returned from the API will now be sorted by the Resource ID (visible in the response) instead of by the Resource's English Label.  I think this makes sense from a consumer perspective even in the response.  From a DB perspective this is _much_ faster because it can just return the clustered index data instead of sorting in memory.

Reply:
I'm not sure the breaking change would be acceptable from our users' perspective. I'm pretty sure at least two different projects are using our endpoint and I'm not sure if they're sorting data on their end. Jesse may have a better read on that. We may in fact want to update to respect the `?Resourcses.SortOrder` column. If we were to add an index on `SortOrder` and `EnglishLabel` would that prevent the in memory sort?

Reply:
It might.  We could try that if this change is unacceptable for consumers.

Reply:
I'm on the fence here. I wouldn't expect a third-party API to handle sorting for me. That's the type of thing that can change without warning, especially if it isn't explicitly documented that you can trust said sorting. If we're seeing a big performance improvement, I'm inclined to just make the change.

Reply:
@jwinston-bn, @MrGrinst, should I go forward with this sorting change?

Reply:
You mean add a sorting parameter that the user can send?